Lib/typing.py

```diff
@@ -725,8 +771,20 @@ def __ror__(self, other):
     def __repr__(self):
         return f'ForwardRef({self.__forward_arg__!r})'
 
-class _TypeVarLike:
-    """Mixin for TypeVar-like types (TypeVar and ParamSpec)."""
+
+def _is_unpacked_typevartuple(x):
+    return (
+            isinstance(x, _UnpackGenericAlias)
+            and isinstance(x.__parameters__[0], TypeVarTuple)
+    )
+
+
+def _is_typevar_like(x):
+    return _is_unpacked_typevartuple or isinstance(x, (TypeVar, ParamSpec))
```

Comment:
```suggestion
    return _is_unpacked_typevartuple(x) or isinstance(x, (TypeVar, ParamSpec))
```
(I'll do a full re-review later)

Reply:
Urk. Well-spotted!